brianlong/www.validators.app
# 182342588 change watchlist foreign key

#### What's this PR do?
- change foreign key in order for dependent: destroy to work correctly on watchlists

#### How should this be manually tested?
- run migrations
- run tests
- try to create user with watchlist and make sure that watchlist_element is deleted if user is

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182342588)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
